google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/gaxx/retrying/ServerStreamingAttemptCallable.java

```diff
@@ -221,12 +222,17 @@ public void onResponseImpl(ResponseT response) {
 
           @Override
           public void onErrorImpl(Throwable t) {
+            ((LargeReadRowsResumptionStrategy) resumptionStrategy).setLargeRowKey(t);
```

Comment:
add a check. If resumptionStrategy is largeRowResumptionStrategty then set the largeRow key.